package.json

```diff
@@ -143,6 +143,7 @@
     "react-transform-catch-errors": "^1.0.2",
     "react-transform-hmr": "^1.0.4",
     "reactable": "^0.14.0",
+    "recompose": "^0.21.2",
```

Comment:
I added this because it makes doing some performance optimizations easier and we can import specific functions from it, so it isn't too bloated.